prospector.go

```diff
@@ -104,6 +105,18 @@ func (p *Prospector) scan(path string, output chan *FileEvent, resume *Prospecto
 			continue
 		}
 
+		// If the file is in the list of exclusions, skip it
+		for _, pattern := range exclude_paths {
+			matched, match_err := filepath.Match(pattern, file)
+			if match_err != nil {
+				emit("Exclusion filepath match error: %v", match_err)
+			} else if matched {
+				emit("\tSkipping exclusion: %s\n", file)
```

Comment:
Not sure on the leading tab here maybe remove it. It might cause parsing issues of syslog if someone is feeding these events into logstash. Nothing else prefixes the tab.